package.json

```diff
@@ -124,6 +124,7 @@
     "test": "make test",
     "test-client": "NODE_ENV=test NODE_PATH=test:client test/runner.js",
     "test-server": "NODE_ENV=test NODE_PATH=test:server:client test/runner.js",
+    "test-tests": "NODE_ENV=test NODE_PATH=test:test/test test/runner.js",
```

Comment:
>But like I said, if you think it's worth changing, I'll be happy to do so.

Nah, I don't think it's worth the fuss. It caught my attention but is a minor detail.